csharp/src/SeedLang/Engine.cs

```diff
@@ -119,6 +124,18 @@ public class Engine {
       }
     }
 
+    // Continues execution of current program. The execution can be stopped from the callback
+    // function of visualization events.
+    public bool Continue(DiagnosticCollection collection = null) {
```

Comment:
Besides the `Continue` interface, a `Stop` or `Quit` interface is required too I think. If a visualizer pauses (via IVM.Stop()) the execution, it will have two choices: (1) continue the execution. (2) stop (quit from) the execution immediately. Or, we can also provide only `Continue` here and provide two interfaces in the IVM interface: `Pause` and `Quit`.

Reply:
Add `Stop()` functions in `Engine` class and `IVM` interface.